t/mojo/json_xs.t

```diff
@@ -72,4 +72,9 @@ like encode_json({test => -sin(9**9**9)}), qr/^{"test":".*"}$/,
 # "escape_slash"
 is_deeply encode_json('/test/123'), '"\/test\/123"', 'escaped slash';
 
+# "allow_dupkeys"
+my $decoded;
+ok eval { $decoded = decode_json('{"test":1,"test":2}'); 1 }, 'no duplicate keys error';
```

Comment:
Why `eval`?

Reply:
The test failure condition (allow_dupkeys not being set) throws an exception.

Reply:
Then why not let it throw? Think this is inconsistent with other tests.

Reply:
Removed the eval